Bukkit/src/main/java/com/plotsquared/bukkit/schematic/StateWrapper.java

```diff
@@ -57,6 +59,10 @@ public class StateWrapper {
     public org.bukkit.block.BlockState state = null;
     public CompoundTag tag = null;
 
+    /**
+     * @deprecated in favour of using WE methods for obtaining NBT
```

Comment:
```suggestion
     * @deprecated in favour of using WE methods for obtaining NBT
```
For sake of completeness we should link a method or a link to the docs outlining a method to use.